src/lib/OpenEXRUtil/ImfUtilExport.h

```diff
@@ -3,15 +3,86 @@
 // Copyright (c) Contributors to the OpenEXR Project.
 //
 
+#ifndef INCLUDED_IMFUTILEXPORT_H
+#define INCLUDED_IMFUTILEXPORT_H
+
+/// \addtogroup ExportMacros
+/// @{
+
+// are we making a DLL under windows (might be msvc or mingw or others)
 #if defined(OPENEXR_DLL)
-    #if defined(OPENEXRUTIL_EXPORTS)
-        #define IMFUTIL_EXPORT __declspec(dllexport)
-        #define IMFUTIL_EXPORT_CONST extern __declspec(dllexport)
-    #else
-        #define IMFUTIL_EXPORT __declspec(dllimport)
-        #define IMFUTIL_EXPORT_CONST extern __declspec(dllimport)
-    #endif
-#else
-    #define IMFUTIL_EXPORT
-    #define IMFUTIL_EXPORT_CONST extern const
+
+#  if defined(OPENEXRUTIL_EXPORTS)
```

Comment:
Per earlier comments, is there any reason to not just use OPENEXR_EXPORTS? Does OpenEXRUtil really need its own distinct macros?